config/default

```diff
@@ -19,6 +19,10 @@ export elasticsearch_host = localhost
 export elasticsearch_port = 9200
 export elasticsearch_index = c2corg
 
+export redis_url = redis://localhost:6379/
```

Comment:
Thanks to Kombu, you can simply change the url in that case:

http://docs.celeryproject.org/projects/kombu/en/latest/userguide/connections.html#urls